src/utils/db_connector.py

```diff
@@ -97,15 +104,15 @@ def db_import(self):
                     os.system(cmd)
                     i += 1
                 except pd.errors.EmptyDataError:
-                    logging.info("[database] Skipping empty file: %s" % file)
+                    console_warning("[database] Skipping empty file: %s" % file)
```

Comment:
Since we're already (optionally) allowing 2 parameters to all of our other logging helpers, we might as well do the same in `console_warning()`. See my suggestion in utils.py that implements this.
```suggestion
                    console_warning("database", "Skipping empty file: %s" % file)
```